cms/envs/common.py

```diff
@@ -2214,6 +2214,10 @@
 # in the blockstore-based XBlock runtime
 XBLOCK_RUNTIME_V2_EPHEMERAL_DATA_CACHE = 'default'
 
+# Blockstore data could contain S3 links, so this should be lower than Blockstore's
+# AWS_QUERYSTRING_EXPIRE (defaults to 3600)
+BLOCKSTORE_VALUE_CACHE_TIMEOUT = 169200
```

Comment:
The word `VALUE` doesn't add much here; how about `BLOCKSTORE_BUNDLE_CACHE_TIMEOUT` or `BLOCKSTORE_CACHE_TIMEOUT`?

Reply:
yeah naming is always a problem :smile: basically this is going to affect any kind of blockstore information, but as it's being used inside the `BundleCache` I think `BLOCKSTORE_BUNDLE_CACHE_TIMEOUT` would be a better option, changing it now